### PR TITLE
feat: Add more fields to limit-count plugin

### LINKED ISSUE #2301: FR - limit-count plugin form need to add allow_degradation section.
# FR - limit-count plugin form need to add allow_degradation section.

## Apisix Dashboard Plugin Editor
resim

On the limit-count editor there's something missing on form table. 

## Our problem

allow_degradation default value has false. But if we want trying to change it to true there's no option to do that in form table. And if we'll switch to json table and can change it to manually , unfortunatelly  if there will be change on form table after this step it going to dissapear.

## Request

There should be some options on form table be able to do change values which on plugin editors(limit-count , limit-conn etc.)

### PR DESCRIPTION
**Pull request by oil-oil**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Added group, allow_degradation, show_limit_quota_header fields to the limit-count plugin form.

image

**Related issues**

resolve #2301

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by oil-oil** [comment]
🧐 I can pass the test case in the development environment, can you help me re-execute the CI. @bzp2010

File: web/src/components/Plugin/UI/limit-count.tsx [hunk 1]
```diff
@@ -323,6 +323,35 @@ const LimitCount: React.FC<Props> = ({ form, schema }) => {
...
+        initialValue={properties.allow_degradation.default}
+        label="allow_degradation"
+        name="allow_degradation"
+        tooltip={formatMessage({
+          id: 'component.pluginForm.limit-count.allow_degradation.tooltip',
+        })}
+        valuePropName="checked"
+      >
+        <Switch defaultChecked={properties.allow_degradation.default} />
+      </Form.Item>
```
Comment (Baoyuantop) [inline web/src/components/Plugin/UI/limit-count.tsx:343]: why `initialValue` and `defaultChecked` both exist?

Comment (oil-oil) [reply web/src/components/Plugin/UI/limit-count.tsx:343]: > why `initialValue` and `defaultChecked` both exist?

Sorry i forgot to delete the extra one.

**Review by zaunist — APPROVED** [review]
That's great.

**Review by LiteSun — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**